lib/dotcom_web/live/schedule_finder_live.ex

```diff
@@ -62,6 +62,15 @@ defmodule DotcomWeb.ScheduleFinderLive do
       stop_id={@stop.id}
       upcoming_departures={@upcoming_departures |> Enum.take(5)}
     />
+    <.async_result :let={departures} :if={@stop} assign={@departures}>
+      <:loading></:loading>
+      <:failed :let={fail}>
+        <.error_container title={inspect(fail)}>
+          {~t"There was a problem loading schedules"}
+        </.error_container>
+      </:failed>
+      <.remaining_service route_type={@route.type} end_of_service={end_of_service(departures)} />
+    </.async_result>
```

Comment:
Looking at the [Async Operations docs](https://hexdocs.pm/phoenix_live_view/1.1.13/Phoenix.LiveView.html#module-async-operations),  I think the following should work instead of needing the `<.async_result>` component:

```elixir
~H"""
<.remaining_service :if={departures = @departures.ok? && @departures.result}  route_type={@route.type} end_of_service={end_of_service(departures)} />
"""
```